### PR TITLE
Add ubuntu 16.04 box

### PR DESCRIPTION
**Pull request by njam**
Reference: https://github.com/boxcutter/ubuntu/blob/bd63a105badb2938fd02b3b315851fcd0798b844/ubuntu.json

@kris-lab @ppp0 fyi

### CODE REVIEW

**Comment by njam** [comment]
@ppp0 please review

I'm still waiting for AWS support to create a "default VPC" for our AWS account, to build this box on "m4.large" (which only runs in a VPC). Once that's done I can create the boxes.

File: ubuntu-1604/default.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        "passwd/username=vagrant ",
+        "<enter>"
+      ],
+      "boot_wait": "3s",
+      "disk_size": 30000,
+      "guest_os_type": "Ubuntu_64",
+      "http_directory": "http",
+      "iso_checksum": "fe495d34188a9568c8d166efc5898d22",
+      "iso_checksum_type": "md5",
+      "iso_url": "http://ch.archive.ubuntu.com/ubuntu/dists/xenial/main/installer-amd64/current/images/netboot/mini.iso",
```
Comment (ppp0) [inline ubuntu-1604/default.json]: * iso_url generic, ie no `ch.` prefix

File: ubuntu-1604/default.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        "passwd/user-password-again=vagrant ",
+        "passwd/username=vagrant ",
+        "<enter>"
+      ],
+      "boot_wait": "3s",
+      "disk_size": 30000,
+      "guest_os_type": "Ubuntu_64",
+      "http_directory": "http",
+      "iso_checksum": "fe495d34188a9568c8d166efc5898d22",
+      "iso_checksum_type": "md5",
```
Comment (ppp0) [inline ubuntu-1604/default.json]: screen shot 2016-05-09 at 12 21 59

There's a bit too many different checksum types for my taste.. How about use `sha256` from now on? (Debian8 already is, the others can stay.. future distros with `sha-256`? wdyt?

File: ubuntu-1604/plain.json [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        "passwd/user-password-again=vagrant ",
+        "passwd/username=vagrant ",
+        "<enter>"
+      ],
+      "boot_wait": "3s",
+      "disk_size": 30000,
+      "guest_os_type": "Ubuntu_64",
+      "http_directory": "http",
+      "iso_checksum": "fe495d34188a9568c8d166efc5898d22",
+      "iso_checksum_type": "md5",
```
Comment (ppp0) [inline ubuntu-1604/plain.json]: checksum_type and iso_url: ditto, see default.json

**Comment by ppp0** [comment]
lgtm, see comments

**Comment by njam** [comment]
Done.

Waiting on AWS support now.

**Comment by njam** [comment]
Ok it's done, boxes released.
@ppp0 please review

**Comment by ppp0** [comment]
Looks good!

**Pull request merged**